source/hardware/controllers/denon_mc7000.rst

```diff
@@ -31,39 +25,23 @@ Features
 -  XLR Booth and Master connections
 
 Compatibility
--------------
+~~~~~~~~~~~~~
 
 -  **Mac** users should be just fine connecting the MC7000 and go.
--  **Windows** users need to install the Windows Driver from `Denon
-   Download Site <https://www.denondj.com/downloads>`__.
+-  **Windows** users need to install the latest Windows Driver from `Denon Download Site <https://www.denondj.com/downloads>`__.
 -  **Linux** users need to know that the MC7000 internal audio interface
-   is not available out-of-the-box for older Linux Kernels. You should
+   is not available out-of-the-box for older Linux Kernels. You have to
    upgrade your **Kernel** to minimum versions LTS: **4.19.105** or
-   **5.4.21**, stable branch **5.6.x** or mainline **5.7.x** (valid at
-   date 2020-04-28). Newer Kernels will also provide native audio
-   support for this controller. Linux Kernel 5.3.x does not support that
-   device and together with 5.5.x reached End-Of-Life.
-
-   -  Linux Distributions built upon Ubuntu 18.04 and derivatives, like
-      Linux Mint, KDE Neon etc. must update the Kernel to one of the
-      mentioned above. As of July 2020 the Kernel 5.4.x is available
-      from the Ubuntu 18.04 Update repo… Get latest upgrades with “sudo
-      apt dist-upgrade”. Alternatively, there is a `Kernel update
-      script <https://github.com/pimlie/ubuntu-mainline-kernel.sh>`__
-      available that helps fetching and installing different Linux
-      Kernels from the Ubuntu Kernel PPA.
-   -  Ubuntu 20.04 already comes with Kernel 5.4.x so it supports the
-      MC7000 Audio Interface out-of-the-box. A fresh install is
-      recommended but you can read further how to `upgrade Ubuntu 18.04
-      to
-      20.04 <https://ubuntu.com/tutorials/tutorial-upgrading-ubuntu-desktop#1-before-you-start>`__.
-   -  OpenSUSE Tumbleweed is currently on Kernel 5.6.x supporting the
-      MC7000 Audio Interface.
-   -  Manjaro Linux 19.0 and 20.0 feature Linux Kernel 5.4.x supporting
-      the MC7000 Audio Interface.
-
+   **5.4.21**. All kernel versions **5.6.x** and following are supporting the device. 
+   All recent Linux distributions are shipped with Audio support for the MC7000, like
+        -  Ubuntu 20.04 and 20.10
```

Comment:
unexpected indentation.

Reply:
I cannot see what is wrong...

Reply:
In RST, you can just indent by a random number of spaces.